cartoframes/vis/defaults.py

```diff
@@ -12,16 +12,16 @@
 _AIRSHIP_ICONS_PATH = 'https://libs.cartocdn.com/airship-icons/v2.0/icons.css'
 
 _CREDENTIALS = {
-  'username': 'default',
-  'api_key': 'default_public',
-  'base_url': ''
+    'username': 'default',
```

Comment:
Should we use a valid carto-run user account here? E.g., we could use `cartoframes` (https://cartoframes.carto.com)

Reply:
OK